src/main/java/uk/gov/companieshouse/api/testdata/model/rest/ResolutionDescriptionType.java

```diff
@@ -0,0 +1,111 @@
+package uk.gov.companieshouse.api.testdata.model.rest;
+
+import com.fasterxml.jackson.annotation.JsonValue;
+
+public enum ResolutionDescriptionType {
+    ELECTIVE_RESOLUTION("elective-resolution"),
+    EXTRAORDINARY_RESOLUTION("extraordinary-resolution"),
+    EXTRAORDINARY_RESOLUTION_ACQUISITION("extraordinary-resolution-acquisition"),
+    EXTRAORDINARY_RESOLUTION_ADOPT_MEMORANDUM("extraordinary-resolution-adopt-memorandum"),
+    EXTRAORDINARY_RESOLUTION_ALTERATION_MEMORANDUM("extraordinary-resolution-alteration-memorandum"),
+    EXTRAORDINARY_RESOLUTION_CAPITALISATION("extraordinary-resolution-capitalisation"),
+    EXTRAORDINARY_RESOLUTION_EXEMPTION("extraordinary-resolution-exemption"),
+    EXTRAORDINARY_RESOLUTION_INCREASE_CAPITAL("extraordinary-resolution-increase-capital"),
+    EXTRAORDINARY_RESOLUTION_MEMORANDUM("extraordinary-resolution-memorandum"),
+    EXTRAORDINARY_RESOLUTION_PURCHASE_NUMBER_SHARES("extraordinary-resolution-purchase-number-shares"),
+    EXTRAORDINARY_RESOLUTION_PURCHASE_SHARES("extraordinary-resolution-purchase-shares"),
+    EXTRAORDINARY_RESOLUTION_REDUCTION_CAPITAL("extraordinary-resolution-reduction-capital"),
+    EXTRAORDINARY_RESOLUTION_REMOVAL_PRE_EMPTION("extraordinary-resolution-removal-pre-emption"),
+    EXTRAORDINARY_RESOLUTION_SECURITIES("extraordinary-resolution-securities"),
+    EXTRAORDINARY_RESOLUTION_VARYING_SHARE_RIGHTS("extraordinary-resolution-varying-share-rights"),
+    LEGACY("legacy"),
```

Comment:
LEGACY("legacy") - does not appear in api-enumerations filing_history_descriptions.yml